include/revng/Model/Register.h

```diff
@@ -66,6 +76,15 @@ type: enum
   - name: r12_arm
   - name: r13_arm
   - name: r14_arm
+  - name: r15_arm
+  - name: q0_arm
+  - name: q1_arm
+  - name: q2_arm
+  - name: q3_arm
+  - name: q4_arm
+  - name: q5_arm
+  - name: q6_arm
+  - name: q7_arm
```

Comment:
What are these? Some ISA extension?

Reply:
The q-aliases were added with VFPv3, those are technically optional, but I think most modern ARM cpus implement them, here's the mapping (from the doc):

> The mapping between the registers is as follows:

> `S<2n>` maps to the least significant half of `D<n>`.
> `S<2n+1>` maps to the most significant half of `D<n>`.
> `D<2n>` maps to the least significant half of `Q<n>`.
> `D<2n+1>` maps to the most significant half of `Q<n>`.

Reply:
Uhm, wait are these aliases for other registers we already mention?
We don't put aliases here but only the largest version.

Reply:
So `q0` is the name representing the whole 64-bit register?
If so, looks good!